nav2_behavior_tree/plugins/control/recovery_node.cpp

```diff
@@ -83,6 +96,7 @@ BT::NodeStatus RecoveryNode::tick()
           {
             // halt second child, increment recovery count, and tick first child in next iteration
             ControlNode::haltChild(1);
+            last_recovery_time_ = rclcpp::Clock().now();
```

Comment:
If the node is halted it doesn't mean there was a recovery.. Maybe set it to 0 or something